events/lifestyles/wc_lifestyle_events.txt

```diff
@@ -0,0 +1,31 @@
+﻿#Events for Mystics
```

Comment:
wdym "Events for Mystics" ?

Reply:
Cleared in 8da51cc1803a3b0c29c377cbe76c316ba9013218.